### DISCUSSION TITLE
Apiman 2.1.4.Final is out

### DISCUSSION THREAD

**Discussion by msavy** [apiman/apiman#1708] [Announcements]
Hi again, everyone.

I've released Apiman 2.1.4.Final. 

This is mainly to provide some new security updates to Netty (relevant for those of you using the Apiman Vert.x. gateway).

https://github.com/apiman/apiman/releases/tag/2.1.4.Final

**Comment by iamakashk** [comment]
Hi @msavy 

It seems like log4j v2.16.0 is vulnerable. Please check this link. 
https://www.zdnet.com/article/apache-releases-new-2-17-0-patch-for-log4j-to-solve-denial-of-service-vulnerability/?ftag=COS-05-10aaa0g&taid=61bf5000f25af2000197c8e8&utm_campaign=trueAnthem%3A+Trending+Content&utm_medium=trueAnthem&utm_source=twitter

Are we going to update to 2.17.0 ?

**Reply by msavy** [reply]
Yes, will cut a release today.

**Reply by msavy** [reply]
Release is done and should be in Maven Central (and all other places) by the time you read this.
